.github/workflows/application-signals-e2e-test.yml

```diff
@@ -76,5 +76,13 @@ jobs:
       adot-image-name: ${{ inputs.adot-image-name }}
       caller-workflow-name: 'main-build'
 
+  python-ecs-e2e-test:
+    uses: aws-observability/aws-application-signals-test-framework/.github/workflows/python-ecs-test.yml@ecs_entry
```

Comment:
What does `@ecs_entry` mean?

Reply:
Sorry, it's test branch, should be @main